monitoring/uss_qualifier/scripts/test_docker.sh

```diff
@@ -10,6 +10,8 @@ echo "Ensure the environment is clean"
 echo "============="
 docker-compose -f docker-compose_qualifier_mocks.yaml -p $project_name down
 
+echo "Rebuild images without cache"
+docker-compose -f docker-compose_qualifier_mocks.yaml -p $project_name build --no-cache
```

Comment:
This will make running tests locally take a lot longer in general -- could we add this as an option that can be triggered when needed, or perhaps just put it in the documentation?

Reply:
Sure, I added a check for the `$CI` environment variable which is [automatically populated by the github actions](https://docs.github.com/en/actions/learn-github-actions/environment-variables)